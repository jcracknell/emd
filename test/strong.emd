**strong text**

strong **across
lines**

no**space**

missing **end
