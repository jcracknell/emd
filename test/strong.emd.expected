#! emd.cs.Conversion.StructuralRenderer, emd.cs
(Document
	(Paragraph
		(Strong
			(Text "strong")
			(Space)
			(Text "text")
		)
	)
	(Paragraph
		(Text "strong")
		(Space)
		(Strong
			(Text "across")
			(Space)
			(Text "lines")
		)
	)		
	(Paragraph
		(Text "no")
		(Strong
			(Text "space")
		)
	)
	(Paragraph
		(Text "missing")
		(Space)
		(Strong
			(Text "end")
		)
	)	
)
